C++/2. Variables.md

```diff
@@ -6,27 +6,27 @@ Variables are containers which can hold values of a specific type. They are word
 
 - `bool`
 - `char`
-- `short`
-- `unsigned`
+- `short int`
 - `int`
-- `long`
+- `unsigned int`
+- `long int`
 - `float`
-- `long long`
+- `long long int`
 - `double`
 
 ## Explanation
 
 The `bool` type means a boolean, which can only be `true` or `false`. The values it can be are those itself.
-The `char` type means a (ASCII) character. `char` stands for character, which means one number or letter. `char`s are encased in single quotes ''.
-The `short` type means a short integer. It can usually hold integers as smaller than `int`, and may take less space.
+The `char` type means a (ASCII) character. `char` stands for character, which means one digit, letter, or symbol. `char` literals are encased in single quotes `''` (`'z'`).
+The `unsigned int` type means an unsigned integer. Unsigned refers to how it lacks a sign, as in a `+` or `-` sign. For example, `4` is a valid value of `unsigned int`, whereas `-4` is not. It can hold (slightly) higher positive values, yet no negatives.
+The `short int` type means a short integer. It can usually hold integers as smaller than `int`, and may take less space.
 The `int` variable type means an integer, which means a whole number.
```

Comment:
`int` comes before `unsigned int` and `short int` in the list.

Reply:
oh wait `short int` comes before, but `unsigned int` and `long int` come after.

Reply:
I see, I don't like my order anymore, I'll fix all sections' orders at once.

Reply:
Updated the file, can you check it out?